go/interpreter/lfvm/opcode.go

```diff
@@ -444,6 +444,9 @@ func (o OpCode) HasArgument() bool {
 		return true
 	case PUSH1_PUSH4_DUP3:
 		return true
+	case PC:
+		return true
```

Comment:
Can you explain this to me? 
Why wasn't it needed before?

Reply:
it's not that it was not needed, perhaps it was simply not noticed that it was missing. 
This should be here since when the code is converted the position where the instruction is called, is saved as argument. Hence when printing the converted code (sole use of `HasArgument`) we want to also show the value of this position.